src/processor/operator/ddl/add_property.cpp

```diff
@@ -0,0 +1,73 @@
+#include "processor/operator/ddl/add_property.h"
+
+#include "storage/in_mem_storage_structure/in_mem_column.h"
+
+namespace kuzu {
+namespace processor {
+
+void AddProperty::executeDDLInternal() {
```

Comment:
My general comment on AddProperty operator is that we should hide the logic of creating files, filling default values into the storage side (NodesStore, RelsStore, NodeTable, RelTable), so we can better separate storage and query processor.

Reply:
Moved the logic of creating files to storageUtils